services/dag-indexer/src/mappings/utils.ts

```diff
@@ -0,0 +1,69 @@
+import { decodeNode, IPLDNodeData, PBNode } from '@autonomys/auto-dag-data'
+
+export const PAD_ZEROS = '0'.repeat(32)
+
+export const hexToUint8Array = (hex: string): Uint8Array => {
+  if (hex.length % 2 !== 0)
+    throw new Error('Hex string must have an even length')
+  return new Uint8Array(
+    hex.match(/.{1,2}/g)?.map((byte) => parseInt(byte, 16)) || [],
+  )
+}
+
+export const getSortId = (
+  blockHeight: bigint | string,
+  indexInBlock?: bigint | string,
+): string => {
+  const str1 = blockHeight.toString().padStart(32, PAD_ZEROS)
```

Comment:
You can get away with padding 10 digits here (or 20 if you want to be future proof for u64):
https://github.com/autonomys/subspace/blob/5e1bdf06578f060cfc46754c90ca3f5c4fbab78b/crates/subspace-core-primitives/src/lib.rs#L123
https://github.com/autonomys/subspace/blob/5e1bdf06578f060cfc46754c90ca3f5c4fbab78b/crates/sp-domains/src/bundle.rs#L415